features/testbot/zero-speed-updates.feature

```diff
@@ -111,8 +111,8 @@ Feature: Check zero speed updates
             """
 
         When I route I should get
-          | from | to | bearings | code    |
-          |    1 |  2 | 270 270  | NoRoute |
```

Comment:
yes, just missed the point that was testing on bicycle profile that allows  routing in reverse direction, for testbot profile it is not allowed